contributor-growth/docs/recruiting-playbook.md

```diff
@@ -0,0 +1,88 @@
+
+For the purpose of this document, we will dig into the how and where to do outreach [TODO other retention related docs and info; contributor ladder]
+
+# Who
+Outreach depends on your audience, how they consume information, and their motivations to help an open source project.  [TODO add much more about motivations of people doing open source - maybe cncf survey related]
+
+1. Downstream community members
+   [TODO add more here, motivations, where to reach, etc]
+   - Companies using your project
+   - Individuals
+   - CNCF End User Committee 
+2. Ecosystem (and dependacy) contributors and maintainers
+   [TODO add more here, motivations, where to reach, etc]
+   - People who build on top of, with, for extendability and any other reason
+3. Academia - students, professors/administrators, scientists, interns
+   [TODO add more here, motivations, where to reach, etc]
+5. TODO who else?
+
+# Where
+Always be recruiting. Think of ways to always promote your needs and areas of growth for the project anytime you have a chance. Be as specific as you can when you are communicating though. The more ambiguous it sounds, the less likely someone is to check it out or take interest. Talk about roles (example: reviewers in a certain area/subject, community managers, project managers, go developers) and provide any kind of guides, role descriptions, contributor ladders as you have. 
+
+General areas with specifics below:
+- your repo via labels, issues calling out help wanted areas, project boards, and other organization
+- a contributor site/area of the project site for contributors
+- mailing list adverts and promotions (weekly help wanted?)
+- in-person or virtual talks and workshops about the project
+- section during community meeting 
+- project twitter announcements 
+
+# Specific Projects & Programs 
+
+## Contributor Documentation
+Contributor Guide (for all contributors)
+[TODO include recommendations and links]
+Developer Guide (deep dive for developers)
+[TODO include recommendations and links]
+
+
+## Issue and PR Triage strategy 
+This is the first thing new contributors will see after some kind of interest in your project.
+Good first issues
+Help Wanted
+Be kind.
+In triage and anywhere in your community. if folks are coming around to file issues and submit one-off PRs, being kind can help them want to do it again the next time they think about it. Saying please, thank you, and welcome never hurts. 
+
+
+## New Contributor Workshops
+[TODO add summary]
+
+### At targeted events
+[TODO - add example links to documentation and sites, etc]
+
+### Online 
+[TODO - add example links to documentation and sites, etc.] 
```

Comment:
- code walk through videos!
- links to talks in conferences.
- links to presentations/meetups

Reply:
Added to a commit, thanks @dims  !